src/components/batch-selection/batch-selection-test.stories.tsx

```diff
@@ -42,3 +42,22 @@ Default.args = {
   selectedCount: 0,
   colorTheme: "transparent",
 };
+
+export const BatchSelectionComponent = ({
+  children,
+  selectedCount = 0,
+  ...args
```

Comment:
This is minor so happy to accept this as it is, but to avoid potential confusion with storybook functionality, it might be worth renaming the remaining props like so: 

```suggestion
  ...rest
```